src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
-                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]\t");
+      out.append(msg.timestamp() + "\t"
+                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]: ");
       return out;
    }
 
 
    // helper for normal
    std::string normalToString(const LogMessage& msg) {
       auto out = LogDetailsToString(msg);
-      out.append('"' + msg.message() + '"');
+      out.append(msg.message() + '\n');
       return out;
    }
 
    // helper for fatal signal
    std::string  fatalSignalToString(const LogMessage& msg) {
       std::string out; // clear any previous text and formatting
-      out.append("\n" + msg.timestamp()
+      out.append(msg.timestamp()
                  + "\n\n***** FATAL SIGNAL RECEIVED ******* \n"
-                 + '"' + msg.message() + '"');
```

Comment:
1. Yes, I agree it is easy to customize. But the default format should work out of box. For example, the case that user mixed with std::cout and demo ColorCoutSink  doesn't get along well with each other would lead to bad user experiences. That's why I strongly against add new line at the beginning of message.